test/e2e/redirect_port_stripping_test.go

```diff
@@ -0,0 +1,139 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"bufio"
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
+)
+
+func TestSecureRedirectStripsPort(t *testing.T) {
+	t.Parallel()
+
+	kubeConfig, err := config.GetConfig()
+	if err != nil {
+		t.Fatalf("failed to get kube config: %v", err)
+	}
+	kclient, err := client.New(kubeConfig, client.Options{})
+	if err != nil {
+		t.Fatalf("failed to create runtime client: %v", err)
+	}
+	kubeClient, err := kubernetes.NewForConfig(kubeConfig)
+	if err != nil {
+		t.Fatalf("failed to create kube client: %v", err)
+	}
```

Comment:
There is already a global `kclient` set up for the whole test suite: [link](https://github.com/openshift/cluster-ingress-operator/blob/b62dd27c901d495e6e78b11968872c002cc2416f/test/e2e/operator_test.go#L120). Is there a reason you need a dedicated one?

Reply:
Nope, we don't, ty. The test will now use the global kclient.